localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```diff
@@ -138,11 +138,8 @@
  innovation_night_elf_ground:0 "Darnassian Ground Tactics"
  innovation_night_elf_ground_desc:0 "With the help of our formidable craftsmanship and blacksmithing skills, we are able to craft extraordinary weapons and armor for our Watcher regiments and have even equipped the mighty Mountain Giants. These primeval giants were created by the Titans from solid stone as guardians of nature itself and endowed with incredible physical and magical resistance."
 
- innovation_pandaren_ground:0 "Pandaren Ground Tactics"
- innovation_pandaren_ground_desc:0 "With our cultural training techniques and traditions, we could develop regiments like the mighty Bearpaws and the agile Wardancers. They are paragons among Pandaren, valuing peace despite their skills at war."
-
- innovation_peacekeeper:0 "Peacekeepers"
- innovation_peacekeeper_desc:0 "The peacekeepers defend us from outside threats with all their might. When needed, these great fighters will wage our wars with unwavering spirit. Peacekeepers are better equipped than most of our soldiers in order for them to overcome the challenges we have to face."
```

Comment:
When we remove lines, we leave empty lines so lines below keep their line number.